projects/table-builder/src/lib/components/header-menu/header-menu.component.ts

```diff
@@ -67,5 +71,20 @@ export class HeaderMenuComponent {
     event.stopPropagation();
     event.preventDefault();
   }
+  checkedItems = [];
+  selectChanged($event, val) {

```

Comment:
Maybe a more descriptive name?

Reply:
selected?

Reply:
What was selected?

Reply:
What would you suggest it be changed to?